app/src/main/java/com/cornellappdev/coffee_chats_android/respositories/UserRepository.kt

```diff
@@ -0,0 +1,55 @@
+package com.cornellappdev.coffee_chats_android.respositories
```

Comment:
Should be in the ViewModel instead; you mentioned in our chat you wanted to change it as so, wasn't sure if that was meant for this PR or the next

Reply:
Yep, I'm splitting this into a ViewModel and a Repository in my next PR.